### ISSUE TITLE
Using 2018 import does not work

### ISSUE THREAD

**Issue by hellow554** [AltSysrq/proptest#110]
If I solely do `use proptest::proptest;` and then create a testcase with the `proptest! { ... }` macro, the compiler will error out with:

```
error: cannot find macro `proptest_helper!` in this scope
   --> src/logicbit/logicvector.rs:209:5
    |
209 | /     proptest! {
210 | |         #[test]
211 | |         fn ctor_width(width in 0u8..) {
212 | |             let v = LogicVector::with_width(width as usize);
213 | |             assert_eq!(width, v.width());
214 | |         }
215 | |     }
    | |_____^
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
```

**Comment by Centril** [comment]
cc https://github.com/rust-lang/rust/pull/56053

**Comment by svenstaro** [comment]
I use this and it works for me:

```rust
    use proptest::{prop_assert, prop_assert_eq, prop_assume, proptest, proptest_helper};
```

**Comment by hellow554** [comment]
It's okay for me to include everything I use, but importing `proptest_helper` doesn't seem appropriate to me.

**Comment by svenstaro** [comment]
Yeah, I agree that the ergonomics should be improved on 2018 but at least it's workable for now.

**Comment by AltSysrq** [comment]
This has been fixed in 0.9.0.
